### ISSUE TITLE
Wrong warning about specifying '-y' for microdnf

### ISSUE THREAD

**Issue by soenkeliebau** [hadolint/hadolint#998]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
When using microdnf to install software in a Dockerfile and omitting the '-y' parameter Hadolint warns about specifying it to bypass any user questions.

However for microdnf (as opposed to dnf) the [default](https://github.com/rpm-software-management/microdnf/pull/94) is to not ask, so this parameter is not needed for microdnf.
Hadolint should not throw a warning about this command in a Dockerfile

### Actual behavior
Hadolint warns about the missing '-y' parameter.

![image](https://github.com/hadolint/hadolint/assets/1070361/f3f63747-f822-4dea-940c-bc8ff69beacb)

### Steps to reproduce the behavior

Run hadolint on a Dockerfile with a microdnf command (example given below).

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

Can't directly run the exact version due to this being run as part of a github action with reviewdog, version pulled in is specified here:

https://github.com/reviewdog/action-hadolint/blob/v1.36.0/action.yml#L48

```Dockerfile
RUN microdnf update --setopt=install_weak_deps=0 && \
    microdnf install tar gzip zip kubectl && \
    microdnf clean all
```

**Additional environment details (OS, stack version, etc.)**
This is running as a github action with the following version of the action:

```yaml
  hadolint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # tag=v4.1.1
      - uses: reviewdog/action-hadolint@f7d29ec6dd89022747bd18b5a151b62ac2f1555e # v1.36.0

```

**Comment by choosen** [comment]
I had to apply `-y` to make it work in Fedora ruby 3.3 image from AWS lambda
